src/gateway.c

```diff
@@ -115,17 +115,17 @@ void gateway__close(struct gateway *g)
 	}
 
 /* Encode the given success response and invoke the request callback */
-#define SUCCESS(LOWER, UPPER)                                                  \
+#define SUCCESS_V0(LOWER, UPPER)                                               \
```

Comment:
On the first pass it wasn't clear that the ` _V0` suffix is there to indicate the schema version, it could use some clarification.

Why don't we have a `SUCCESS_V1`?